org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/servlet/ApiRestApplication.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.rest.servlet;
```

Comment:
I would suggest to use org.eclipse.scout.rt.rest without servlet as package name since RestApplication is not directly a servlet and base class "Application" does also not belong to a servlet package.